site/content/integrate/apps/using-third-party-api/_index.md

```diff
@@ -64,7 +63,7 @@ Hello OAuth2! is an HTTP app, it requests the *permissions* to act as a System A
 
 ### Bindings and locations
 
-The Hello OAuth2! creates three commands: `/helloworld configure|connect|send`.
+The Hello OAuth2! creates three commands: `/helloworld configure | connect | send`.
```

Comment:
Should this be called "The Hello OAuth2 _app_"? The exclamation point looks a bit off in the middle of a sentence too